### ISSUE TITLE
Descriptive Analyses

### ISSUE THREAD

**Issue by jzhang722** [jzhang722/Denmark-Immigrant-Health#2]
In this issue, we will discuss potential early descriptive analyses in our exploration of documenting and explaining a immigrant vs native gap in take-up of prescription drugs.

**Comment by jzhang722** [comment]
Here is what I would like to see in terms of early descriptives:

**Raw means (fraction prescribed) for the most common drugs prescribed in Denmark, for immigrants and natives. Specifically:**
- For the top 10 most prescribed drugs classes (e.g., you want fluoxetine and sertraline to count as one "antidepressant" drug), report the fraction of each population that are prescribed that in a given year (immigrant vs native)
   - We want to do this for 10 most prescribed to natives, _and_ 10 most prescribed to immigrants because the overlap of the two might be very different
   - Depending on what these top 10 drugs are, we will want to manually include a couple drugs of specific drug classes (e.g., MH drugs may not make the top 10)
- All ages, and then separately for 0-18, 18-65, and 65+
- Might be easier to just code immigrant as foreign-born?
- Let's focus on a relatively short and recent time period (eg 2016-2018?) What are your thoughts?
- _Maybe later_ but not now: adjust for age, gender etc or adjusting for comorbidities/diagnoses

Meltem and Mircea, let me know if you're getting this without me explicitly tagging you with the "@" symbol.

**Comment by tramir** [comment]
Thanks @jzhang722 for putting this together! A couple of very quick thoughts/notes for the future.

> **Raw means (fraction prescribed) for the most common drugs prescribed in Denmark, for immigrants and natives. Specifically:**
>  * For the top 10 most prescribed drugs classes (e.g., you want fluoxetine and sertraline to count as one "antidepressant" drug), report the fraction of each population that are prescribed that in a given year (immigrant vs native)

Sounds good to me. We might want to extend to top-20, just in case. For the future -- this would mean level-3 or level-4  ATC codes. We have information at level 4 in the variable `atc4` and, for some drug classes, level 5 in the variable `atc`.

>  * Depending on what these top 10 drugs are, we will want to manually include a couple drugs of specific drug classes (e.g., MH drugs may not make the top 10)

Agreed. We might want to specifically target "culturally-sensitive" drugs: MH (most of the drugs in the `N` class), contraceptive pill (@mdaysal knows the codes), and antibiotics. Anything else?

>  * Might be easier to just code immigrant as foreign-born for this exercise?

This is what DST (aka Statistics Denmark) calls "first-generation immigrants." And @mdaysal and I agree that having one native parent is likely to make you more "adapted" than a pure immigrant person.

>  * Let's focus on a relatively short and recent time period (eg 2016-2018?) What are your thoughts?

Let's see how massive the data is. Denmark is not a big country, so we might be able to increase the time span a little. But I agree, we can start with a more manageable data set and expand later.

> Meltem and Mircea, let me know if you're getting this without me explicitly tagging you with the "@" symbol.

As you can see, I do :blush:

**Comment by jzhang722** [comment]
@tramir wow are you sure you've never used Github? You're such a pro with the quoting and everything!

> Sounds good to me. We might want to extend to top-20, just in case. For the future -- this would mean level-3 or level-4 ATC codes. We have information at level 4 in the variable atc4 and, for some drug classes, level 5 in the variable atc.

20 is even better!

> Agreed. We might want to specifically target "culturally-sensitive" drugs: MH (most of the drugs in the N class), contraceptive pill (@mdaysal knows the codes), and antibiotics. Anything else?

MH, oral contraceptive, antibiotics all look good to me. Why not throw in opioids and benzos in too?

> Let's see how massive the data is. Denmark is not a big country, so we might be able to increase the time span a little. But I agree, we can start with a more manageable data set and expand later.

I'm fine with including more years, but then we may want to do some time trend adjusting if immigrants into Denmark look very different from year to year.

One other thing, we might want to break down by gender too.

**Comment by jzhang722** [comment]
@soniabhalotra We just noticed your comments! Thanks! 

Although we were confused because you edited my post (as opposed to responding), and most importantly, none of us were notified. In the future, perhaps we should keep to responding.

**Comment by soniabhalotra** [comment]
thanks Jonathan and thanks everyone for meeting today. I am using the next open box, hoping this is what "responding" is on GitHub. I read over No2(comment), it looks good. I have nothing new to add except that we should break down by the native and immigrant groups by SES; and include antibiotics, painkillers, AD, HRT, contraceptives. I wondered- what is the Statistics Denmark def of immigrants? Sonia

**Comment by soniabhalotra** [comment]
Dear all
I just responded, using a new box. Did you get an alert independent of this email.
Soon, I will learnt the github tricks! Sonia

From: Jonathan Zhang ***@***.***>
Date: Monday, 13 November 2023 at 16:41
To: jzhang722/Denmark-Immigrant-Health ***@***.***>
Cc: Bhalotra, Sonia ***@***.***>, Mention ***@***.***>
Subject: Re: [jzhang722/Denmark-Immigrant-Health] Descriptive Analyses (Issue #2)

Hi all,
I've made some updates to the descriptive task in #2 (comment)<https://github.com/jzhang722/Denmark-Immigrant-Health/issues/2#issuecomment-1790997087> based on our conversation on the call today. Please look it over and make additional comments (e.g., how are we defining "drug classes"; how many digits, etc.)

I know @tramir<https://github.com/tramir> had also mentioned looking at most common diagnoses. Can we spell this out for Angelina? (e.g., outpatient diagnoses only or also hospitalizations?) Feel free to add other tasks please! We should finalize this by end of day Wednesday.

I think we also want to know sample sizes based on various immigration definitions, or should we skip this for now?

**Comment by Lina011** [comment]
Hi all, it was great to see you today! I just would like to add here the definitions of immigrants and descendants from the Statistics Denmark website:
“An immigrant is defined as a person born abroad whose parents are both (or one of them if there is no available information on the other parent) foreign citizens or were both born abroad.”
“A descendant is defined as a person born in Denmark whose parents (or one of them if there is no available information on the other parent) are either immigrants or descendants with foreign citizenship. If there is no available information on either of the parents and the person in question is a foreign citizen, the person is also defined as a descendant.”

**Comment by soniabhalotra** [comment]
Thank you Angelina

From: Lina011 ***@***.***>
Date: Monday, 13 November 2023 at 17:45
To: jzhang722/Denmark-Immigrant-Health ***@***.***>
Cc: Bhalotra, Sonia ***@***.***>, Mention ***@***.***>
Subject: Re: [jzhang722/Denmark-Immigrant-Health] Descriptive Analyses (Issue #2)

Hi all, it was great to see you today! I just would like to add here the definitions of immigrants and descendants from the Statistics Denmark website:
“An immigrant is defined as a person born abroad whose parents are both (or one of them if there is no available information on the other parent) foreign citizens or were both born abroad.”
“A descendant is defined as a person born in Denmark whose parents (or one of them if there is no available information on the other parent) are either immigrants or descendants with foreign citizenship. If there is no available information on either of the parents and the person in question is a foreign citizen, the person is also defined as a descendant.”

**Comment by mdaysal** [comment]
There are two ways to think about common conditions. One, which we mainly focused on is by looking at prescription drugs. This is natural when we want to consider physician prescribing tendencies. These are derived from the prescription register. We know the identifier for the clinic/hospital that wrote the prescription. However, we cannot separate whether it originated from an inpatient/outpatient/ER visit. A second way to think about conditions most relevant for these populations is by looking at distributions of diagnosis codes from the hospital discharge data sets. Here, we have information on ICD-8 (until 1994) and ICD-10 (thereafter) codes for the primary reason for contact as well as codes for secondary conditions. Somatic hospital data dates back to 1981 but outpatient and ER data were added only in 1995. Psychiatric data (inpatient/outpatient/ER) all start in 1995. I think these are not that helpful at this stage -- when we focus on the link between physician prescribing tendency and patient uptake. 

For looking at most common medications, we should stick to ATC code level 3. This should be general enough to capture different diagnoses and thus would also be somewhat informative of prevailing diagnoses. 

I suggest using more detailed age groups: 0-17, 18-24, 25-44, 45-64, 65+

As a recap: we want to get the top-20 medications by immigration status. In addition, Sonia wants to see the fraction of the following:
* antibiotics - J01
* painkillers - N02 Analgesics
* antidepressants - N06A
* HRT - G03C, G03D, G03F, G03XC, G02BA03
* contraceptives - Combined contraceptives (G03AA01, G03AA03, G03AA05, G03AA07, G03AA09, G03AA10, G03AA11, G03AA12, G03AB03, G03AB04, G03AB05, G03AB06, G03HB) + Progestin-only contraception (G03AC01, G03AC02, G03AC03, G03AC09) and the progestin-only implant (G03AC08).

For sanity check: we can use this paper for antibiotics. https://bmcinfectdis.biomedcentral.com/articles/10.1186/s12879-019-3964-9

**Comment by jzhang722** [comment]
@mdaysal I agree with you with everything you wrote on prescriptions, more detailed age groups, and top 20 medications. Only potential concern is that it can become a lot to look at.

As for diagnoses, I think it will be better to not use prescription-based diagnosis measures. First, I don't have any experience with prescription register diagnoses. But more substantively, we want a measure of diagnosis gaps that are not contaminated by prescription immigrant gaps. In other words, if immigrants are less likely to use SSRIs, we want a measure of depression diagnosis that is not a function of SSRI use. My vote would be to use outpatient + inpatient diagnoses. Inpatient by itself will only pick up more severe cases. We can look at the standard Elixhauser or Charlston comorbidities. But I am also 100% happy to look at prescriptions first before diagnoses.

**Comment by mdaysal** [comment]
@jzhang722 I agree that we want to get a measure of diagnosis gaps that are not tainted by treatment decisions. The main issue is that hospital contacts (including outpatient care) will only be relevant for more severe presentations of health issues. For example, most adult mental health care is provided in GP clinics (where we don't have diagnosis information). Similarly, almost all of contraceptives will be prescribed by GPs. Maybe it would be better to say that we want to focus on "treatment gaps" instead of diagnosis gaps. We could then try to link this gap to GP practice style as well as patient characteristics.

**Comment by mdaysal** [comment]
That said, @Lina011 can also create descriptive statistics on outpatient hospital visits. Here we should take a decision on how we want to classify ICD codes. Do we want to look at the top-20 conditions using the most detailed ICD codes or do we want to aggregate them somehow? @jzhang722 you have more experience with hospital data. Do you have any suggestions?

**Comment by jzhang722** [comment]
Oh interesting. I did not know we did not have diagnosis codes for GP visits. For [ICD10 codes](https://www.icd10data.com/ICD10CM/Codes), the first letter tells you the category. But perhaps we can do prescriptions first and circle back to diagnoses in 2 weeks?

**Comment by jzhang722** [comment]
@Lina011 do these steps make sense? Do you have any questions? 

@tramir @mdaysal when is our next meeting? Shall we schedule something? Maybe after @Lina011 gives us an anticipated date she might have something?

**Comment by tramir** [comment]
Angelina has already produced some output last week, but I was able to export it only just now. She'll send us a more "readable" version in a little bit.

**Comment by Lina011** [comment]
Hi everyone @jzhang722 @mdaysal @tramir @soniabhalotra, I am sorry for the delay, I was trying to match the descriptives to the names of the drugs to have a better understanding of the stats. However, it was not easy to find a downloadable version of ICD codes online, both versions that me and @tramir found were not detailed enough to match ATC3 codes. I found a good list on the German official website and used that one, I checked it, and it should be correct. So I attached here two files: (1) top-20 drugs by age group and immigration status as discussed above and (2) fractions by antibiotics, painkillers, antidepressants, HRT, and contraceptives. I also added the paper that @mdaysal shared in the chat during the last meeting. @tramir Thank you for your help with exporting the files.
Let me know if you would like to schedule a meeting to discuss these descriptives. 
[kristiansen_sheng_doctors_ses.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13417939/kristiansen_sheng_doctors_ses.pdf)
 [table_desc_formatted.xlsx](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13417917/table_desc_formatted.xlsx)
[table_desc_fractions.xlsx](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13417918/table_desc_fractions.xlsx)

**Comment by jzhang722** [comment]
Thanks @Lina011! A few things.

Re ICD code matching: it looks like ATC3 are drug codes and ICD are diagnosis condition codes. I understand that the Danish data has diagnoses with prescriptions but I don't see why you need to match these? Are there other codes that are helpful? For example, I have crosswalks to CCS, SNOMED, etc. but these might be only used in the US..

Can you remind me again what immigrant generation 0, 1, and 2 are?

I'll need some time to digest these results. Shall we schedule a meeting to talk them through?

**Comment by Lina011** [comment]
Sorry for the confusion @jzhang722, there are no diagnosis condition codes in these descriptives, only top-20 drugs. Regarding ATC code merging, I just meant that I merged the information on the drug names (the last column) which was not there before. I found it easier to follow the description of the drug rather than a code since I mainly know well AD-related codes. 

Regarding the immigrant definition: 0 - Native, 1 - 1st gen immigrant (“An immigrant is defined as a person born abroad whose parents are both (or one of them if there is no available information on the other parent) foreign citizens or were both born abroad.”), 2 - 2nd gen immigrant (“A descendant is defined as a person born in Denmark whose parents (or one of them if there is no available information on the other parent) are either immigrants or descendants with foreign citizenship. If there is no available information on either of the parents and the person in question is a foreign citizen, the person is also defined as a descendant.”).

freq_withinGroup is the number of prescriptions of that drug within this group divided by the number of prescriptions within this group. freq_total is the number of prescriptions of this drug divided by the number of all prescriptions.
Hope it helps :)
I am happy to schedule a meeting to talk about these in more detail. When would it work best?

**Comment by jzhang722** [comment]
Thanks @Lina011 for clarifying!

> freq_withinGroup is the number of prescriptions of that drug within this group divided by the number of prescriptions within this group. freq_total is the number of prescriptions of this drug divided by the number of all prescriptions.

Ideally we would like a definition that is proportion of individuals who use this drug over some period (eg 1 year). This definition is at the prescription-level and can be impacted if immigrants fill fewer prescriptions to begin with (denominator becomes smaller). I think this works for now but just wanted to highlight that patient-level stats might be better going forward.

@soniabhalotra @tramir @mdaysal meetings I can do after 8am ET (1pm London, 2pm CPH) every day this week, and every day next week outside of Thursday.

**Comment by mdaysal** [comment]
> Thanks @Lina011 for clarifying!
> 
> > freq_withinGroup is the number of prescriptions of that drug within this group divided by the number of prescriptions within this group. freq_total is the number of prescriptions of this drug divided by the number of all prescriptions.
> 
> Ideally we would like a definition that is proportion of individuals who use this drug over some period (eg 1 year). This definition is at the prescription-level and can be impacted if immigrants fill fewer prescriptions to begin with (denominator becomes smaller). I think this works for now but just wanted to highlight that patient-level stats might be better going forward.
> 
> @soniabhalotra @tramir @mdaysal meetings I can do after 8am ET (1pm London, 2pm CPH) every day this week, and every day next week outside of Thursday.

Thanks, @jzhang722. Both Sonia and I have been sick these past few weeks so we are behind Angelina's work. I very quickly looked at the immigrant-native gaps and abstracting from what you wrote above, it does look like the gap is mainly for "culturally sensitive" drugs. For example, the rates seem higher for antibiotics etc. What I suggest as the next step is to create variables of "ever use drug X between ages X-X" and "number of prescriptions for drug X between ages X-X" by immigrant status. Here we need to deal with age at immigration, I think. Do you think this could be a reasonable next step?

**Comment by jzhang722** [comment]
Thanks @mdaysal 

I think that makes sense. @Lina011, Meltem's comment is pretty much echoing my comment earlier that we want to move to patient-level summary statistics. For each of the drug classes in table_desc_fractions (we don't need to look at all the drugs in the other table, but maybe can we include antipsychotics too?), can we compute:

1. Fraction of patients who ever use each drug class (per year)
2. Number of drugs each patient fills of that drug class (per year)

Repeat this for the same age groups and immigrant/native distinctions you did earlier. The (per year) part deals with the fact that immigrants will not have been in the country every year. So for a 12 year old native Dane, you would say in the last 12 years, what fraction of years did they fill an antibiotic? For a 12 year old immigrant who landed at age 10, you would say in the last 2 years, what fraction of years did they fill an antibiotic. This would be a crude but easy adjustment. Let me know if anyone thinks of a better way than this (per year) method.

**Comment by Lina011** [comment]
Thank you both, @mdaysal and @jzhang722. Sorry, I am a bit slow on this as I am also going through job applications at the moment.

I totally agree about looking into the stats per patient per year.
I am just confused about “the number of drugs each patient fills of that drug class per year”.  Do we want to look at the drug types within each drug class? For example, the number of SSRIs prescriptions within the antidepressant class? Or do we simply want the number of antidepressant prescriptions per patient per year?

Also, I am a bit confused about the time span and looking at the fraction of years you were mentioning. Currently, we are looking at the period of 2015-2019 and it would be hard to go back much further in time. This is because the LMDB data on prescriptions are available from 1995-2022 only, so we could only look at children and young adults to have the entire history of prescriptions (which I think could be very interesting on its own). Please correct me if I am wrong @mdaysal @tramir. In addition, I am afraid that looking more than 5 years back for all age categories would increase computational time a lot (we have more than 5 mln individuals per year). 

Perhaps, I could start by generating the stats for each patient regarding:
- “ever using drug X in 2015-2019” by age group and immigration status.
- “the number of prescriptions of drug X per year” by age group and immigration status.
A variation of these stats could be, as @mdaysal proposed: "ever used drug X between ages X-X" and "number of prescriptions for drug X between ages X-X" by immigrant status. 
- “the fraction of patients who ever use each drug class” per year and by immigration status and age group.

To note, I do not have the age of immigration currently in my sample. I could add it to the data and estimate the average age of immigration for each age group of the 1st gen immigrants. Would you agree with these steps?

**Comment by jzhang722** [comment]
> I am just confused about “the number of drugs each patient fills of that drug class per year”. Do we want to look at the drug types within each drug class? For example, the number of SSRIs prescriptions within the antidepressant class? Or do we simply want the number of antidepressant prescriptions per patient per year?

The latter. Call all drugs within the antidepressant class an antidepressant and then count number of those in a given year. Not sure how refills or days supply are handled in Danish data, but let's ignore days supply and count refills. So if I fill 3 Zoloft and 4 Prozacs in one year, that's 7 antidepressants.

> Also, I am a bit confused about the time span and looking at the fraction of years you were mentioning. Currently, we are looking at the period of 2015-2019 and it would be hard to go back much further in time. This is because the LMDB data on prescriptions are available from 1995-2022 only, so we could only look at children and young adults to have the entire history of prescriptions (which I think could be very interesting on its own). Please correct me if I am wrong @mdaysal @tramir. In addition, I am afraid that looking more than 5 years back for all age categories would increase computational time a lot (we have more than 5 mln individuals per year).

That makes sense. You know the data well so we'll leave it up to you to make the executive decisions. If an immigrant lands in 2017, then you want to get the average over two years (2018, 2019) whereas for a native Dane it would be over all years 2015-2019. This is all I meant when I said "per year" to have a common comparison period as opposed to sum over the entire period.

> To note, I do not have the age of immigration currently in my sample. I could add it to the data and estimate the average age of immigration for each age group of the 1st gen immigrants. Would you agree with these steps?

I want to make sure that you are doing this at the individual-level first (e.g., a child who arrives in 2015 is treated differently than a child who arrives in 2018). We only want to average over the years they are in Denmark. Other than that I agree with the three proposed steps you listed!!

**Comment by Lina011** [comment]
Hi all, I am very sorry for the delay on this. I am planning to get it ready this week and will update you as soon as it is ready.
Thank you very much, @jzhang722, for your comments and clarifications!

**Comment by soniabhalotra** [comment]
hello all. I was reading the correspondence as it evolved. I have just read the whole thing again. (1) I agree that we need patient-level statistics. (2) Ideally we would have GP diagnosis and then be able to look at treatment-gaps for each group ie treatment conditional on diagnosis. As we do not have GP diagnosis, I think for now we could start with prescriptions as a measure of treatment. Later on we can look at hospital admissions as another measure of treatment. We will have to assume that immigrants do not have better health than natives so if they are treated more this may be because their health is worse but if they are treated less then this is likely because there are supply or demand barriers to their obtaining treatment. Do you all agree on this. 

Please feel free to disagree with me but my impression is that we should get a first set of descriptive stats under a simpler set of conditions. One complication I would take out at this early stage is age. I am inclined to start by looking at immigrant adults who have been in the country at least 5 years. (Reason: I don't have a hypothesis for why treatment for immigrant children should be any different than for adults). This way we can avoid the complications of looking at the 12 year old who arrived 2 years ago differently from the 15 year old who arrived one year ago. If you agree with my suggestion we can summarise prescription use for immigrants and adults over the same calendar time.

**Comment by soniabhalotra** [comment]
Also thinking of how to simplify the outcomes we want: how about just 2 variables: the extensive and intensive margins: (1 if ever used) and (volume of drug use). if this is obtained at the individual level and then collapsed by (native vs immigrant) then this gives the fraction so in your comment above Jonathan I wasn't sure why fraction was a fourth outcome.

**Comment by jzhang722** [comment]
@soniabhalotra 

> As we do not have GP diagnosis, I think for now we could start with prescriptions as a measure of treatment. Later on we can look at hospital admissions as another measure of treatment. We will have to assume that immigrants do not have better health than natives so if they are treated more this may be because their health is worse but if they are treated less then this is likely because there are supply or demand barriers to their obtaining treatment. Do you all agree on this.

I agree we will need to do some sort of risk adjustment using diagnosis or something. I'm less sure we need to look at treatment. I thought we were originally interested in prescribing differences. I guess we do need to know if they are not being prescribed because they are treated in non-medication ways (e.g., psychotherapy vs pharmacotherapy), so in that sense I agree--but not until later?

> Also thinking of how to simplify the outcomes we want: how about just 2 variables: the extensive and intensive margins: (1 if ever used) and (volume of drug use). if this is obtained at the individual level and then collapsed by (native vs immigrant) then this gives the fraction so in your comment above Jonathan I wasn't sure why fraction was a fourth outcome.

Yes the fraction can be obtained from the other outcomes directly. I'm fine with simplifying too. @Lina011 I'll let you make the executive decisions--the key is to make sure every comparison is as apples-to-apples as possible. As for removing age, I'm indifferent. But since Angelina already has age breakdowns from earlier, maybe we stick to keeping the age heterogeneity for now? Many of the medications are age-specific.

**Comment by Lina011** [comment]
Hi all, sorry for the delay, but it took some time to figure out how to extract the information from different atc level codes (atc, atc2, atc3) for each patient within the drug family.

I attach here the updated ratios of patients who filled for a given drug family, in a given year. I prepared graphs for each drug family, split by age groups over the years. I think it is easier to see the differences between natives and immigrants this way.
Also, there are two sets of results: (1) baseline, everyone who is available in the data, irrespective of their year of arrival; and (2) marked with immbefore2015 includes only immigrants who are in the country for the entire observed period 2015-2019 or last 5 years.
I also prepared the dataset with the number of prescriptions/ever-using drugs per patient per year (not sharing it here). There are around 5 mln patients so this is a very large table. I also added the year and age of migration for the 1st generation of immigrants to this table, but I need to spend a bit more time thinking about how to use that in the most meaningful way. 
One proposition could be restricting per age group, e.g. for 0-17 to include only immigrants who are there for 17 years, 18-24 who are there for 24, and so on. What do you think? 
Another concern is that I am not sure we can look into the volume of the drug used, we can see the number of prescriptions, but we do not know the prescribed dosage.

The differences in prescriptions by immigrant status mostly come from the different age groups. E.g. within one age group, the largest differences in prescriptions are observed for contraceptives and antidepressants across natives and immigrants. Also noticeable, antibiotics are most prescribed to migrants in the age group 45-64.

I am afraid I will be a bit slow on this till January as I need to put more focus on the job market paper at the moment but looking forward to exploring these results in detail and adding hospital admissions.

[table_desc_patient_all.xlsx](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664367/table_desc_patient_all.xlsx)
[table_desc_patient_antibiotics.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664368/table_desc_patient_antibiotics.pdf)
[table_desc_patient_antibiotics_immbefore2015.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664369/table_desc_patient_antibiotics_immbefore2015.pdf)
[table_desc_patient_antidepressants.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664370/table_desc_patient_antidepressants.pdf)
[table_desc_patient_antidepressants_immbefore2015.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664371/table_desc_patient_antidepressants_immbefore2015.pdf)
[table_desc_patient_combined_contr.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664372/table_desc_patient_combined_contr.pdf)
[table_desc_patient_combined_contr_immbefore2015.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664373/table_desc_patient_combined_contr_immbefore2015.pdf)
[table_desc_patient_hrt.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664374/table_desc_patient_hrt.pdf)
[table_desc_patient_hrt_immbefore2015.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664375/table_desc_patient_hrt_immbefore2015.pdf)
[table_desc_patient_immbefore2015.xlsx](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664376/table_desc_patient_immbefore2015.xlsx)
[table_desc_patient_p_only_contr.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664377/table_desc_patient_p_only_contr.pdf)
[table_desc_patient_p_only_contr_immbefore2015.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664378/table_desc_patient_p_only_contr_immbefore2015.pdf)
[table_desc_patient_p_only_implant.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664379/table_desc_patient_p_only_implant.pdf)
[table_desc_patient_p_only_implant_immbefore2015.pdf](https://github.com/jzhang722/Denmark-Immigrant-Health/files/13664380/table_desc_patient_p_only_implant_immbefore2015.pdf)

**Comment by mdaysal** [comment]
Hi all,

We fly to Princeton Friday morning, my last lecture is still tomorrow, no luggages ready and we’ve all got the flu. That’s why we are MIA. We’ll get back to this and act like proper co-authors once we are all settled. Thanks for your patience!

Meltem
--
N. Meltem Daysal
Associate professor
Department of Economics
University of Copenhagen
Denmark
Email: ***@***.***
Website: www.meltemdaysal.com<http://www.meltemdaysal.com/>

On Dec 13, 2023, at 20:18, Jonathan Zhang ***@***.***> wrote:

﻿

Thanks @Lina011<https://github.com/Lina011>

A few questions:

  *   combined_contr: is all forms of female contraceptive? Is the denominator restricted to female only or both sexes?
  *   p_only_contr and p_only_implant: what are these and how do they relate to combined_contr?

My interpretation:
There's a lot of large and neat differences.
Antibiotics are most common among younger immigrants. My first hypothesis was that perhaps the immigrant kids are less healthy, but this doesn't explain how 2nd gen is high and even higher than 1st gen.
Antidepressants, contraceptives, and HRT are entirely consistent with our priors: these things are foreign to immigrants. Large differences by native vs immigrant for antidepressants and contraceptives for young adults!
Overall the first vs second generation immigrants are in weird directions: first generation looks more like natives than second generation... anyone surprised by this?

**Comment by soniabhalotra** [comment]
Dear Mm
I hope you recover before flying
Lots of good luck with this important visit!


Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by soniabhalotra** [comment]
Thanks for bringing us back Jonathan!
How did your job market go? Where will you go?
I’m ready when you are

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jzhang722** [comment]
@soniabhalotra I officially signed with Duke Sanford school of public policy this morning! January was tough, but so relieved it's over now.

**Comment by mdaysal** [comment]
> @soniabhalotra I officially signed with Duke Sanford school of public policy this morning! January was tough, but so relieved it's over now.

Congrats!!!! I heard that it was a very tough market this year. I am also happy to get back to it. I understand from Mircea that your data access is still not solved. It would be good to sort it out soon as US universities cannot get access to the data from Rockwool. @tramir If this may be a problem when Jonathan moves to Duke, maybe we can start a new project on KU server? Or check if we can piggyback on one of my already approved projects.

**Comment by tramir** [comment]
Congratulations @jzhang722! This is great news and Duke is, as far as I know, a nice place. Hope it all works out really well. As a related aside: as @mdaysal wrote above, this makes data access for you trickier because the US is not a country that plays well with GDPR. But we'll get this sorted out and set up once you are officially in Duke. Hopefully the legal office will not give us as much trouble as the one at McMaster :)

For the project: this is indeed a good time for us to go back to it, but let's also hear back from @Lina011's about how her JM is going and when she can get back to it.

**Comment by jzhang722** [comment]
> Congrats!!!! I heard that it was a very tough market this year. I am also happy to get back to it. I understand from Mircea that your data access is still not solved. It would be good to sort it out soon as US universities cannot get access to the data from Rockwool. @tramir If this may be a problem when Jonathan moves to Duke, maybe we can start a new project on KU server? Or check if we can piggyback on one of my already approved projects.

I just followed up on that again and reply-all'ed the Rockwool email. I am officially "on leave" for a year from McMaster so I will look like I am formally affiliated with a non-US institution until July 1 2025

**Comment by mdaysal** [comment]
OK that should make things easy in the short-run!

**Comment by soniabhalotra** [comment]
Congrats Jonathan. My co-author Joanna Maselko (epi) spent many years at Duke before moving to UNC and her husband Manoj Mohanan (global health) is still there. They invited me a couple of times but, because of childcare, I never made it there. Just in case you don’t know anyone there yet, I can put you in touch with them as they are really nice caring people.

On data access- is there a risk that Jonathan will simply not get access? This would be very disappointing.

From: Mircea Trandafir ***@***.***>
Date: Friday, 9 February 2024 at 20:43
To: jzhang722/Denmark-Immigrant-Health ***@***.***>
Cc: Bhalotra, Sonia ***@***.***>, Mention ***@***.***>
Subject: Re: [jzhang722/Denmark-Immigrant-Health] Descriptive Analyses (Issue #2)

Congratulations @jzhang722<https://github.com/jzhang722>! This is great news and Duke is, as far as I know, a nice place. Hope it all works out really well. As a related aside: as @mdaysal<https://github.com/mdaysal> wrote above, this makes data access for you trickier because the US is not a country that plays well with GDPR. But we'll get this sorted out and set up once you are officially in Duke. Hopefully the legal office will not give us as much trouble as the one at McMaster :)

For the project: this is indeed a good time for us to go back to it, but let's also hear back from @Lina011<https://github.com/Lina011>'s about how her JM is going and when she can get back to it.

**Comment by jzhang722** [comment]
@soniabhalotra what a small world! Manoj is the associate dean so he's the one I've been communicating with the most. I didn't know his wife was Joanna who is on your PPD paper. Very cool!

**Comment by tramir** [comment]
The world is so small! I know Manoj as well and I was thinking of mentioning him, but it seemed like he was in a different school. I don't know Joanna, but Manoj is a really nice person.

**Comment by Lina011** [comment]
Hi all, 

Happy to hear from you all! Congratulations, @jzhang722, on your new position, this is great news!

Would it be fine with you all to touchbase next week, please? I am quite flexible next week, Tuesday to Friday works best, except Wed 12-2pm and Thursday 1-2pm and 3-4pm UK time.
This week is quite busy for me, so it would be better if we could start discussing the project next week. Hope it works for you all.

**Comment by jzhang722** [comment]
Most of these times work for me, so I'll leave it to the busier people. I (and probably @mdaysal @tramir) can only do after 1pm UK time though.

**Comment by mdaysal** [comment]
I have a seminar on Feb 21 and our daughter’s school is closing early/opening late twice this week and fully closed on Feb 19. Could we resume the meetings after Feb 21? If possible, having a recurring meeting schedule on the same day/time would be highly preferable for me.
--
N. Meltem Daysal
Associate professor
Department of Economics
University of Copenhagen
Denmark
Email: ***@***.***
Website: www.meltemdaysal.com<http://www.meltemdaysal.com/>

On Feb 12, 2024, at 08:35, Jonathan Zhang ***@***.***> wrote:

﻿

Most of these times work for me, so I'll leave it to the busier people. I (and probably @mdaysal<https://github.com/mdaysal> @tramir<https://github.com/tramir>) can only do after 1pm UK time though.

**Comment by mdaysal** [comment]
@jzhang722 @tramir @Lina011 @soniabhalotra Did we settle on a meeting day/time?

**Comment by jzhang722** [comment]
Maybe Thursday mornings reoccurring? Does 10am ET work for people? I'm flexible (not teaching this semester).

**Comment by soniabhalotra** [comment]
I can do Thur thanks

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by mdaysal** [comment]
Thursday 10 ET is usually OK for me but I have a meeting on Feb 22. We could meet at 10:30 on Feb 22 and then switch to weekly meetings at 10:00 if it works for everyone.
--
N. Meltem Daysal
Associate professor
Department of Economics
University of Copenhagen
Denmark
Website: www.meltemdaysal.com

On 15 Feb 2024, at 15.35, soniabhalotra ***@***.***> wrote:


I can do Thur thanks

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by tramir** [comment]
Thursdays at 10 works for this semester. We might need to revisit after we return to Denmark because of school pickups...

**Comment by soniabhalotra** [comment]
This isfine with me

From: mdaysal ***@***.***>
Date: Friday, 16 February 2024 at 01:29
To: jzhang722/Denmark-Immigrant-Health ***@***.***>
Cc: Bhalotra, Sonia ***@***.***>, Mention ***@***.***>
Subject: Re: [jzhang722/Denmark-Immigrant-Health] Descriptive Analyses (Issue #2)
Thursday 10 ET is usually OK for me but I have a meeting on Feb 22. We could meet at 10:30 on Feb 22 and then switch to weekly meetings at 10:00 if it works for everyone.
--
N. Meltem Daysal
Associate professor
Department of Economics
University of Copenhagen
Denmark
Website: www.meltemdaysal.com

On 15 Feb 2024, at 15.35, soniabhalotra ***@***.***> wrote:


I can do Thur thanks

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Lina011** [comment]
Hi all, 
Unfortunately, Thursdays 10 does not work well for me because it overlaps with the department seminars. Could we do 10:30 instead (3:30 pm UK)?

I will still schedule a zoom meeting for the next week as we already agreed on 10:30. I will send the invite by email to everyone.

**Comment by mdaysal** [comment]
10:30 is also OK for me.
--
N. Meltem Daysal
Associate professor
Department of Economics
University of Copenhagen
Denmark
Website: www.meltemdaysal.com

On 16 Feb 2024, at 08.56, Lina011 ***@***.***> wrote:



Hi all,
Unfortunately, Thursdays 10 does not work well for me because it overlaps with the department seminars. Could we do 10:30 instead (3:30 pm UK)?

I will still schedule a zoom meeting for the next week as we already agreed on 10:30. I will send the invite by email to everyone.

**Comment by jzhang722** [comment]
10:30 recurring works for me

**Comment by soniabhalotra** [comment]
Me too

From: Jonathan Zhang ***@***.***>
Date: Friday, 16 February 2024 at 14:32
To: jzhang722/Denmark-Immigrant-Health ***@***.***>
Cc: Bhalotra, Sonia ***@***.***>, Mention ***@***.***>
Subject: Re: [jzhang722/Denmark-Immigrant-Health] Descriptive Analyses (Issue #2)

10:30 recurring works for me

**Comment by soniabhalotra** [comment]
Just checking  we meet in 45 min?
I’m in the new flat in leamington and there’s no wifi yet and bad signal do I’ll go to a cafe for our meeting

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jzhang722** [comment]
Yes we meet in 45!
